apis/accelerate-api.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: accelerate-api
+  description: Accelerate springboot REST API
+spec:
+  type: openapi
+  lifecycle: experimental
+  owner: accelerate-team
+  system: accelerate-system
+  definition:
+    $text: https://github.com/FaissalElfid/backstage-manifest/blob/main/api/accelerate.json
+    # $text: https://accelerate-backend.campus.clusterdiali.me/api/v1/hello-octo/api-docs #working on the app cors
```

Comment:
Le backend expose le JSON de l'OpenApi: https://accelerate-backend.campus.clusterdiali.me/api/v1/hello-octo/api-docs
Mais par sécurité le backend bloque le cors donc le backstage ne peux pas fetcher le JSON. (je vais essayer de régler ce problème par la suite)

Reply:
I can help on that